win/WIN_Slider.cpp

```diff
@@ -183,7 +184,7 @@ bool Slider::mouseButtonUp(MouseButton button)
 		int xMouse = 0;
 		int yMouse = 0;
 		
-		SDL_GetMouseState(&xMouse, &yMouse);
+		getRenderer()->getMouseState(xMouse, yMouse);
```

Comment:
Should probably just pass these in as parameters. This violates SRP (Single Responsibility Principle) - a Renderer should just be for rendering, not for getting peripheral state).
**MUST**

Reply:
I put the mouse state function here to separate SDL from Slider (and other classes that rely on mouse states), but there's nothing that requires it be in Renderer. So I've moved getMouseState out of Renderer into win::utils.